README.md

```
@@ -1,23 +1,29 @@
-## Contributing
-See [`CONTRIBUTING.md`](CONTRIBUTING.md) for more information on contributing to CircuitVerse.
+Set the TRANSLATE_API_ENDPOINT environment variable:
+TRANSLATE_API_ENDPOINT=https://your-production-api-endpoint.com/translate
 
-## License
-This project is licensed under the [MIT License](LICENSE).
+or simple use shell to enter 
+
+export TRANSLATE_API_ENDPOINT=https://your-production-api-endpoint.com/translate
+
+# Action Required for production or long-term usage:
```

Markdownlint (0.35.0)

Punctuation: ':'
Trailing punctuation in heading

(MD026, no-trailing-punctuation)